src/lib/client/GoodCityApiV2Client/GoodCityApiV2Client.test.ts

```diff
@@ -0,0 +1,108 @@
+import GoodCityApiV2Client from "./GoodCityApiV2Client";
```

Comment:
The general consensus on naming is that capitalised names refer to Types/Classes/Modules.

Here`GoodCityApiV2Client` is a function, so `goodCityApiV2Client` would be more appropriate.

I would say that the name is very long and verbose, `client` would be my preferred choice personally.